src/admin/openapi.yaml

```diff
@@ -132,6 +135,16 @@ paths:
     $ref: paths/admin@categories@bulk.yaml
   /admin/expense_fields:
     $ref: paths/admin@expense_fields.yaml
```

Comment:
Could you let me know if you are delivering everything together?
if not, please divide this into separate PRs so you can deliver incrementally.

Reply:
Will split into parts, to users it'll be delivered in two releases